src/index.rst

```diff
@@ -148,6 +148,7 @@ The definition of done is used to guide teams in planning and estimating the siz
   development/fundamental_sw_requirements
   development/python-codeguide
   development/javascript-codeguide
+  development/python_package_release_procedure
```

Comment:
Add a section in the homepage with a short description and a link to the relevant page, taking example from the other pages in the same section